checkov/main.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import annotations
 
 import atexit
+from email.policy import default  # noqa: F401
```

Comment:
where do you use this import?

Reply:
I don't use this, seems to be a spurious addition - I have removed it.